js_modules/dagster-ui/packages/ui-core/src/runs/QueuedRunCriteriaDialog.tsx

```diff
@@ -119,7 +123,11 @@ const QueuedRunCriteriaDialogContent = ({run}: {run: RunTableRunFragment}) => {
                         </td>
                         <td>
                           <Tag interactive>
-                            {limit.value !== undefined ? `${limit.key}=${limit.value}` : limit.key}
+                            {limit.value !== undefined
+                              ? isPlainObject(limit)
+                                ? `${limit.key}=${JSON.stringify(limit.value)}`
+                                : `${limit.key}=${limit.value}`
```

Comment:
Wouldn't JSON.stringify result in the same output even if its not a plain object?

Reply:
yea nice